tap/tlstapper/tls_poller.go

```diff
@@ -135,24 +126,12 @@ func (p *tlsPoller) handleTlsChunk(chunk *tlsChunk, extension *api.Extension,
 	key := buildTlsKey(chunk, ip, port)
 	reader, exists := p.readers[key]
 
-	newReader := NewTlsReader(
-		key,
-		func(r *tlsReader) {
-			p.closeReader(key, r)
-		},
-		chunk.isRequest(),
-		p,
-	)
-
 	if !exists {
-		reader = p.startNewTlsReader(chunk, ip, port, key, extension, newReader, options)
+		reader = p.startNewTlsReader(chunk, ip, port, key, emitter, extension, options)
 		p.readers[key] = reader
 	}
 
-	tlsReader := reader.(*tlsReader)
-
-	tlsReader.setCaptureTime(time.Now())
```

Comment:
The place where we call `time.Now()` was there;

https://github.com/up9inc/mizu/commit/b5c665b60236b7868833b2c1d31a7510e1b54ddc#diff-2c7816ba77c385ccf0193c52e07bdc1a012be29c6c0f33389735f77d2a9f1ea2R134

Now it's moved to somewhere else. Is it intended?

Reply:
No, it wasn't.
Thanks, I put it back there.